### PR TITLE
PL 803xx release notes

### PR DESCRIPTION
**Pull request by deepakbolangady**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md [hunk 1]
```diff
@@ -16,6 +16,26 @@ For Harness on-prem releases, go to [Harness Self-Managed Enterprise Edition Rel
 
 If you don't see a new feature or enhancement in your Harness account, it might be behind a Feature Flag. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
 
+## Latest: Version 803xx
+
+#### New features and enhancements
```
Comment (aimurphy) [inline docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md:21]:
```suggestion
### New features and enhancements
```

File: docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md [hunk 2]
```diff
@@ -16,6 +16,26 @@ For Harness on-prem releases, go to [Harness Self-Managed Enterprise Edition Rel
...
+
+- If you use the App Role authentication method in the Hashicorp Vault connector, you can choose to cache the vault token. The token is cached on the Harness Delegate for a time duration equal to the TTL of the vault token, with 1% leeway. 
+
+  By default, caching is enabled for all existing connectors. To disable caching, go to the connector's YAML configuration and set the `enableCache` parameter to `false`. Harness UI support to enable and disable caching will be added in a subsequent release. (PL-39821)
+
+  This item requires Harness Delegate version 803xx. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate). 
+
+#### Early access features
```
Comment (aimurphy) [inline docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md:29]:
```suggestion
### Early access features
```

File: docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md [hunk 3]
```diff
@@ -16,6 +16,26 @@ For Harness on-prem releases, go to [Harness Self-Managed Enterprise Edition Rel
...
+
+This release does not include early access features.
+
+#### Fixed issues
```
Comment (aimurphy) [inline docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md:33]:
```suggestion
### Fixed issues
```

**Review by aimurphy — DISMISSED** [review]
LGTM, just one possible heading level issue.

**Review by SudheendraKatte — DISMISSED** [review]

**Review by SudheendraKatte — APPROVED** [review]

**Pull request merged**